src/features/bulk/store/__stubs__/state.ts

```diff
@@ -0,0 +1,75 @@
+import { CustomFile } from '@amsterdam/bmi-component-library';
+import { IBulkMetadataField, IBulkMetadataFile, IBulkMetadataFileMetadata, IBulkMetadataState } from '../model'
+
+function makeFile(id: string, uploadedFile: CustomFile, metadata: IBulkMetadataFileMetadata[]): IBulkMetadataFile {
+	return {
+		id,
+		url: 'some-url',
+		uploadedFile,
+		metadata
+	}
+}
+function makeCustomFile(id: number, title: string, filetype?: string) {
+	return {
+		tmpId: id,
+		progress: 0,
+		type: filetype ?? 'application/pdf',
```

Comment:
Is there a reason why we use `application/pdf` as default fallback?

Reply:
just for stubs